content/en/technical-debt.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Technical Debt
+status: Feedback Appreciated
+category: property
+---
+
+
+Technical debt is a concept in software development that reflects the implied cost of additional rework caused by choosing an easy solution for now instead of using a better approach that would take longer. It is a part of every software development project since trade-offs between speed, cost, and quality are impossible to avoid. Technical debts could arrive due to various defects in documentation, code, automation, design, architecture, etc. The best practices to counter technical debt include
+1) Establishing proper architecture. Designing a fitting architecture for the project avoids the majority of bugs.
```

Comment:
Not sure we should add best practices here. Explaining what it is, should be enough for a glossary definition. I'd skip this section completely.